### PR TITLE
Redirected artifactory.

### PR DESCRIPTION
**Pull request by andricDu**

### CODE REVIEW

**Comment by andricDu** [comment]
Working :smile:

Now to eliminate those pesky asterisks and possibly update documentation.

File: dcc-portal-ui/bower.json [hunk 1]
```diff
@@ -49,15 +49,15 @@
...
     "x2js": "~1.1.7",
-    "lodash": "3.10.*",
-    "angular-mocks": "~1.4.*",
+    "lodash": "3.10.1",
+    "angular-mocks": "~1.4.11",
     "zeroclipboard": "~2.2.0"
   },
   "devDependencies": {
-    "angular-scenario": "~1.4.*"
+    "angular-scenario": "~1.4.11"
```
Comment (btiernay) [inline dcc-portal-ui/bower.json]: Any danger in removing the `~`?

**Comment by btiernay** [comment]
I agree with this dude: https://gist.github.com/jashkenas/cbd2b088e20279ae2c8e

>The responsible way to upgrade isn't to blindly pull in dependencies and assume that all is well just because a version number says so — the responsible way is to set aside five or ten minutes, every once in a while, to go through and update your dependencies, and make any minor changes that need to be made at that time. If an important security fix happens in a version that also contains a breaking change for your app

**Comment by andricDu** [comment]
I strongly agree. Periodically dedicating some time to increment and very the versions would potentially save lots of headaches.

File: dcc-portal-ui/bower.json [hunk 2]
```diff
@@ -27,34 +27,34 @@
...
+    "angular-ui-router": "0.2.15",
+    "ngInfiniteScroll": "1.2.0",
+    "angular-bootstrap": "0.12.1",
+    "angular-local-storage": "0.1.5",
+    "angularytics": "0.3.0",
     "angular-ui-utils": "bower-scrollfix",
-    "angular-markdown-directive": "~0.3.1",
+    "angular-markdown-directive": "0.3.1",
     "FileSaver": "1.3.1",
     "Blob": "*",
```
Comment (btiernay) [inline dcc-portal-ui/bower.json]: What about this guy!

File: dcc-portal-ui/bower.json [hunk 3]
```diff
@@ -27,34 +27,34 @@
...
-    "angular-animate": "~1.4.11",
-    "angular-cookies": "~1.4.11",
-    "angular-loading-bar": "~0.6.0",
-    "restangular": "~1.4.0",
-    "angular-ui-router": "~0.2.15",
-    "ngInfiniteScroll": "~1.2.0",
-    "angular-bootstrap": "~0.12.1",
-    "angular-local-storage": "~0.1.5",
-    "angularytics": "~0.3.0",
+    "angularjs": "1.4.11",
```
Comment (btiernay) [inline dcc-portal-ui/bower.json]: ^^ `3.8.21.x` should be updated to `4.0.x` No need to update for each point release.

File: dcc-portal-ui/package.json [hunk 4]
```diff
@@ -10,42 +10,42 @@
...
+    "grunt-contrib-imagemin": "0.9.4",
+    "grunt-contrib-jshint": "0.11.3",
+    "grunt-contrib-uglify": "0.11.1",
     "grunt-contrib-watch": "0.6.1",
-    "grunt-filerev": "~2.3.1",
+    "grunt-filerev": "2.3.1",
     "grunt-google-cdn": "0.2.0",
-    "grunt-injector": "~0.6.x",
-    "grunt-jsdoc-to-markdown": "~1.2.1",
+    "grunt-injector": "0.6.x",
```
Comment (btiernay) [inline dcc-portal-ui/package.json]: Should this guy be pinned?

File: dcc-portal-ui/package.json [hunk 5]
```diff
@@ -10,42 +10,42 @@
...
+    "grunt-jsdoc-to-markdown": "1.2.1",
     "grunt-karma": "0.10.1",
-    "grunt-ng-annotate": "~0.9.2",
+    "grunt-ng-annotate": "0.9.2",
     "grunt-open": "0.2.0",
-    "grunt-peg": "~1.5.0",
+    "grunt-peg": "1.5.0",
     "grunt-usemin": "3.0.0",
     "http-proxy": "0.10.0",
     "jasmine-core": "^2.4.1",
```
Comment (btiernay) [inline dcc-portal-ui/package.json]: These are still floating.

File: dcc-portal-ui/bower.json (hunk 2 above)
Comment (andricDu) [inline dcc-portal-ui/bower.json]: If you remember, this guy aint got no version :(

Comment (btiernay) [inline dcc-portal-ui/bower.json]: Doesn't he have an MD5 or something?

File: dcc-portal-ui/package.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "dcc-portal-ui",
   "description": "ICGC DCC Data Portal UI",
-  "version": "3.8.21.x",
+  "version": "4.0.26",
   "license": "MIT",
```
Comment (btiernay) [inline dcc-portal-ui/package.json]: I thought we were GPL?

**Pull request merged**